torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -228,9 +226,13 @@ The explanation needs to be clear on why this is needed. Here are some good exam
       rejection_reason = await this.reasonToRejectForceRequest(forceMessage);
     } else {
       // Ensure the PR has been signed off on
-      let isApprovedPR = await this.doesHaveApprovedStatus()
-      if (!isApprovedPR) {
-        rejection_reason = "This PR needs to be approved by an authorized maintainer before merge."
+      let approval_status = await this.getApprovalStatus()
+      if (approval_status !== 'approved') {
+        if (approval_status === 'changes_requested') {
+          rejection_reason = "There are changes requested on this PR. Please address the request and either get a re-review from the person or dismiss their review."
```

Comment:
One needs admin or write permissions to dismiss a review.
Can we detect the user status to not suggesting dismissing for read-only contributors to avoid confusion?
Or maybe just don't mention dismissing at all?

Reply:
Going back to the old behavior where approvals trump changes requested, so this message is disappearing